packages/components/src/components/select-native/select-native.jsx

```diff
@@ -15,9 +15,27 @@ const getDisplayText = (list_items, value) => {
     return '';
 };
 
-const SelectNativeOptions = ({ list_items, should_hide_disabled_options, use_text }) => {
+const SelectNativeOptions = ({ label, list_items, should_hide_disabled_options, use_text }) => {
     const options = should_hide_disabled_options ? list_items.filter(opt => !opt.disabled) : list_items;
-
+    const grouped = options.reduce((r, a) => {
+        r[a.group] = [...(r[a.group] || []), a];
+        return r;
+    }, {});
+    const keys = Object.keys(grouped);
+    const values = Object.values(grouped);
+    if (label === 'Asset') {
+        return keys.map((option, i) => {
+            return (
```

Comment:
you can omit **return**, it will work anyway

Reply:
Thanks, refactored that line.